docs/guide/installation.md

```
@@ -0,0 +1,165 @@
+  // build: { },   // 传给 builder 插件（可选）
+  render: { baseDir: './' }, // 解析 entryPath / dslPath 时使用的基准目录，一般为项目根
+})
+```
+
+### 根配置 `zelpis`
+
+渲染插件会读取 `config.zelpis`；若缺少该字段，渲染插件在配置阶段会报错。因此必须在 `defineConfig` 里提供 `zelpis`，至少包含 `entrys`。
+
+每个 入口（`entrys` 中一项）常用字段：
+
+| 字段          | 说明
```

markdownlint-cli2 (0.22.1)

[warning] Table pipe style
Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe

(MD055, table-pipe-style)